src/main.ts

```diff
@@ -505,6 +511,11 @@ ipcMain.handle('browser:open:external:link', (_, url: string) => {
   shell.openExternal(url)
 })
 
+ipcMain.handle('application:open-log', () => {
```

Comment:
minor suggestion: should we shorten it with `app:open-log` ?
I'm not sure about this